.pre-commit-config.yaml

```diff
@@ -19,13 +19,12 @@ repos:
   hooks:
   - id: isort
     args:
+    # Keep in sync with pyproject.toml
+    - --force-single-line-imports
```

Comment:
Does this seem like a subjective choice? Why not use tables with mandatory trailing commas?

Reply:
I would also prefer to avoid it and only use options for which we have clear reasoning.

Reply:
the pattern was adopted early on, and the config was intended to minimize the changes, in an attempt to focus enabling isort and less so on the style.

My preference here would be to stay this course since the pattern is in place, and revisit it later.

this comes to mind: https://www.python.org/dev/peps/pep-0008/#imports

Reply:
BTW, I am happy to raise an issue to revisit the isort configuration so it is not lost or forgotten.

Reply:
> this comes to mind: [python.org/dev/peps/pep-0008/#imports](https://www.python.org/dev/peps/pep-0008/#imports)

That section allows `from ... import <multiple things>` which is what I meant. I didn't mean that bare imports should be inlined.

Reply:
Because the current pattern in place is for single line from statements, I'd prefer to keep it that way, revisit later, and focus on getting this in place.  It will eventually be one less thing that needs to be done manually, which frees up time to work on issues and features that effect the user experience.

Reply:
@ssbarnea @webknjaz The line now reads:

force_single_line = true # Force from .. import to be 1 per line, minimizing changes at time of implementation

I see this as resolved and am comfortable with the configuration as it is.

Can you resolve this please?

Reply:
resolving conversation, please reopen if the concerns and question are unanswered, ty for the feedback